relay-server/src/actors/processor.rs

```diff
@@ -520,8 +520,13 @@ impl FromMessage<RateLimitFlushBuckets> for EnvelopeProcessor {
 /// Service implementing the [`EnvelopeProcessor`] interface.
 ///
 /// This service handles messages in a worker pool with configurable concurrency.
+#[derive(Clone)]
 pub struct EnvelopeProcessorService {
     config: Arc<Config>,
```

Comment:
`config` can also move into `inner`. Only the new global config (which is not part of this PR) will be outside of the `Arc`.

Reply:
I wasn't sure why but this is what jan was suggesting when all paired 🤔

Reply:
ok nvm, was a misunderstanding, changed it now